# HG changeset patch
# User Sebastian Wiesner <lunaryorn@googlemail.com>
# Date 1287494849 -7200
# Node ID a6efb65cb62598bf49eacf18fe209c2d92f794f5
# Parent  b09f7601e2bb5163a03439c5e41c61070f5aaf14
Extend module path to find lodgeitlib when uninstalled

diff -r b09f7601e2bb -r a6efb65cb625 doc/conf.py
--- a/doc/conf.py	Sun Sep 12 14:53:03 2010 +0200
+++ b/doc/conf.py	Tue Oct 19 15:27:29 2010 +0200
@@ -1,6 +1,10 @@
 # -*- coding: utf-8 -*-
 
 import sys, os
+
+sys.path.append(
+    os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir)))
+
 import lodgeitlib
 
 
